MAINTAINERS.md

```diff
@@ -0,0 +1,21 @@
+The current maintainers of this repo are [@adamralph](https://github.com/adamralph), [@bording](https://github.com/bording), and [@WojcikMike](https://github.com/WojcikMike).
+
+The maintainers [watch](https://github.com/Particular/NServiceBus.RabbitMQ/watchers) this repo and undertake the following responsibilities:
+
+- Ensuring that the `develop` branch is always releasable. "Releasable" means that the software built from the latest commit can be released immediately simply by following the release procedure.
+  - This does not imply that the latest commit should *always* be released, only that it *can* be, immediately after deciding to release.
+- Releasing new versions of the software.
+- Reviewing and merging [pull requests](https://github.com/Particular/NServiceBus.RabbitMQ/pulls).
+- [Issue backlog](https://github.com/Particular/NServiceBus.RabbitMQ/issues) grooming, including the triage of new issues as soon as possible after they are created.
```

Comment:
Given the other items in this list, to be consistent, this should be "Grooming the issue backlog".